app/views/components/inputCharacterCount.scala.html

```diff
@@ -0,0 +1,52 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@this(govukCharacterCount: GovukCharacterCount)
+
+@(
+        form: Form[_],
+        id: String,
+        name: String,
+        rows: Int,
+        value: Option[String],
+        maxLength: Option[Int],
+        label: String,
+        isPageHeading: Boolean,
+        headingMessageArgs: Seq[String] = Seq(),
+        hint: Option[Html] = None,
+        classes: Option[String] = None,
+        countMessageClasses: String,
```

Comment:
could you please check use of countMessageClasses and stripWhitespace, its defined but not used.